frontends/web/src/locales/en/app.json

```diff
@@ -265,6 +265,11 @@
     },
     "stepCreate": {
       "description": "This name is used as the device name and for the backup.",
+      "error": {
+        "genericMessage": "For maximum compatibility only ASCII characters and spaces are supported, max 30 chars.",
```

Comment:
same as before, please remove 'maximum compatibility', it kind of does not mean anything and just adds work to translators. 

i was wondering if ASCII might not be understandable to a normal uset. chatgpt suggets

> Use letters, numbers, basic symbols, spaces. Max 30 characters.